src/shared/ptyfwd.c

```diff
@@ -490,18 +485,6 @@ int pty_forward_new(
                         tcsetattr(f->input_fd, TCSANOW, &raw_stdin_attr);
                 }
 
-                if (tcgetattr(f->output_fd, &f->saved_stdout_attr) >= 0) {
-                        struct termios raw_stdout_attr;
-
-                        f->saved_stdout = true;
-
-                        raw_stdout_attr = f->saved_stdout_attr;
-                        cfmakeraw(&raw_stdout_attr);
-                        raw_stdout_attr.c_iflag = f->saved_stdout_attr.c_iflag;
-                        raw_stdout_attr.c_lflag = f->saved_stdout_attr.c_lflag;
-                        tcsetattr(f->output_fd, TCSANOW, &raw_stdout_attr);
-                }
```

Comment:
hmm, i have the suspicion we should keept this so that we get raw output, but make sure we then override ONLCR explicitly, so that we get the NL → CR/LF mapping which we probably need.

Also, the code needs fixing in case input fd and output fd actually point to the same tty.

Reply:
lemme play around with this for a moment